pkgs/applications/networking/instant-messengers/matrix-commander/default.nix

```diff
@@ -0,0 +1,38 @@
+{ stdenv, lib, fetchFromGitHub, cacert, python3 }:
+
+stdenv.mkDerivation {
+  pname = "matrix-commander";
+  version = "unstable-2021-04-18";
+
+  src = fetchFromGitHub {
+    owner = "8go";
+    repo = "matrix-commander";
+    rev = "3e89a5f4c98dd191880ae371cc63eb9282d7d91f";
+    sha256 = "08nwwszp1kv5b7bgf6mmfn42slxkyhy98x18xbn4pglc4bj32iql";
+  };
+
+  buildInputs = [
+    cacert
+    (python3.withPackages(ps: with ps; [
+      matrix-nio
+      magic
+      markdown
+      pillow
+      urllib3
+      aiofiles
+    ]))];
+
+  installPhase = ''
+    mkdir -p $out/bin
+    cp $src/matrix-commander.py $out/bin/matrix-commander
+    chmod +x $out/bin/matrix-commander
+  '';
+
+  meta = with lib; {
+    description = "Simple but convenient CLI-based Matrix client app for sending and receiving";
+    homepage = "https://github.com/8go/matrix-commander";
+    license = licenses.gpl3;
```

Comment:
gpl3Plus or gpl3Only?

Reply:
I didn't find any mentions about that in the matrix-commander repo, so I [asked on github](https://github.com/8go/matrix-commander/issues/29), but there's no answer yet. Should wait, infer gpl3Only, or keep gpl3 for now?

Reply:
I think keeping it gpl3 for now and adding a link to the issue is the best way to handle this. Then it's easy to change in the future if / when an answer comes.

Reply:
ok, added a note in [1331ee3](https://github.com/NixOS/nixpkgs/pull/119895/commits/1331ee3fae7930840714173a3bdec79a528ec54b)